### DISCUSSION TITLE
Crazyswarmm simulator dimensions

### DISCUSSION THREAD

**Discussion by Iko-Git** [USC-ACTLab/crazyswarm#443] [Q&A] [answered]
Hello!
The crazyswarm simulator dimensions by default are 
x=[-4,+4]
y=[-4,+4]
z=[0,+3]
is there any way to change the simulator environment dimensions?
Thanks!

**Comment by whoenig** [comment, marked as answer]
Only directly in the code at https://github.com/USC-ACTLab/crazyswarm/blob/5142a433fc6425178845d9cf90f34ab9a397c122/ros_ws/src/crazyswarm/scripts/pycrazyswarm/visualizer/visMatplotlib.py#L13-L15

**Reply by Iko-Git** [reply]
Thanks a lot! @whoenig
